egs/swbd/s5c/local/nnet3/prepare_multistyle_data.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+
+. ./cmd.sh
+
+set -e
+stage=1
+train_stage=-10
+generate_alignments=false
+multi_style=false
+noise_list="reverb:music:noise:babble:clean"
+augment_test_set=false
+suffix=""
+clean_ali=tri4_ali_nodup
+train_set=train_nodup
+
+. ./path.sh
+. ./utils/parse_options.sh
+
+mkdir -p exp/nnet3
+
+if [ -e data/rt03 ]; then maybe_rt03=rt03; else maybe_rt03= ; fi
+
+if [ "$multi_style" == "true" ]; then
```

Comment:
I think we should remove the multi-style option here, for purposes of experiments, since if people didn't want it, they just shouldn't call this.  I know this was useful for you... you can of course use it for your own experiments, but i would have though just settinng noise_list to "clean" would do what you want.
Also I don't like the name "noise_list" as reverb and clean are not noise.  Let's call it "augmentation_list" wherever it occurs.